webserver/views/user.py

```diff
@@ -26,6 +27,17 @@ def lastfmscraper(user_id):
     )
     return Response(scraper, content_type="text/javascript")
 
+@user_bp.route("/<user_id>/resettoken", methods=["GET", "POST"])
+@login_required
+def reset_token(user_id):
+    if request.method == "POST":
+        reset = request.form.get("reset")
+        if reset == "yes":
+            db.user.update_token(current_user.id)
+            flash.info("Access token reset")
```

Comment:
It'll ask you to log in. And then it might reset your token. Note that the user_id isn't actually used, it uses `current_user`, which is based on who you are logged in as.  Perhaps we could remove `<user_id>`